sass/main.scss

```diff
@@ -12,7 +12,16 @@ body {
     font-size: $fontSize-base;
     background-color: $colour-mineshaft;
 }
-
+.jumbotron, .container {
```

Comment:
For future reference (don't worry about it for now), I'd avoid dumping stuff in the main.scss. if these are overrides for bootstrap, we could stick 'em in a _bootstrap-overrides.scss - if there's likely to be lots of these, we could set up a bootstrap folder with individual files in there.

Reply:
Ok